src/components/ResourceBrowser/ResourceList/Sidebar.tsx

```diff
@@ -296,33 +229,38 @@ const Sidebar = ({
             option.groupName,
             option.label !== (SIDEBAR_KEYS.namespaces as Nodes) &&
                 option.label !== (SIDEBAR_KEYS.events as Nodes) &&
-                option.label !== (SIDEBAR_KEYS.nodes as Nodes) &&
-                option.label !== (SIDEBAR_KEYS.overview as Nodes),
+                option.label !== (SIDEBAR_KEYS.nodes as Nodes),
         )
     }
 
-    function formatOptionLabel(option: K8sObjectOptionType, formatOptionLabelMeta: FormatOptionLabelMeta<any>) {
-        return (
-            <div className="flex left column">
-                {!formatOptionLabelMeta.inputValue ? (
-                    <span className="w-100 dc__ellipsis-right">{option.label}</span>
-                ) : (
-                    <span
-                        className="w-100 dc__ellipsis-right"
-                        dangerouslySetInnerHTML={{
-                            __html: option.label.replace(
-                                new RegExp(formatOptionLabelMeta.inputValue, 'gi'),
-                                (highlighted) => `<mark>${highlighted}</mark>`,
-                            ),
-                        }}
-                    />
-                )}
-            </div>
-        )
-    }
+    const formatOptionLabel = useCallback(
+        (option: K8sObjectOptionType, formatOptionLabelMeta: FormatOptionLabelMeta<K8sObjectOptionType>) => {
+            return (
+                <div className="flex left column">
+                    {!formatOptionLabelMeta.inputValue ? (
+                        <span className="w-100 dc__ellipsis-right">{option.label}</span>
+                    ) : (
+                        <span
+                            className="w-100 dc__ellipsis-right"
+                            dangerouslySetInnerHTML={{
+                                __html: option.label.replace(
+                                    new RegExp(formatOptionLabelMeta.inputValue, 'gi'),
+                                    (highlighted) => `<mark>${highlighted}</mark>`,
+                                ),
```

Comment:
Check if we can use highlightSearchText from fe-common and use DomPurify whenever using dangerouslySetInnerHTML